src/components/navigation/navigation-item/NavigationItem.tsx

```diff
@@ -188,14 +190,14 @@ export class NavigationItem extends Component<NavigationItemProps, NavigationIte
   };
 
   render() {
-    const {data, selectedItem, showIcon} = this.props;
+    const {data, selectedItem, showIcon, instructionLabel} = this.props;
     const {id, previewImage, itemType, displayTime, liveCuePoint, groupData, displayTitle, displayDescription} = data;
     const {imageLoaded} = this.state;
     const ariaLabelTitle: string = (typeof displayTitle === 'string' && displayTitle ? displayTitle : displayDescription) || '';
 
     const a11yProps: Record<string, any> = {
       ['aria-current']: selectedItem,
-      ['aria-label']: ariaLabelTitle,
+      ['aria-label']: displayTime + " " + ariaLabelTitle + " " + instructionLabel,
```

Comment:
what if the language is right-to-left ?

Reply:
there is no issue. screen reader will announce the times then aria-label and in the end the instructions.

Reply:
sure, all the accessibility changes (not clear bugs) are discussed with product